From b7c5f14021ff76d5b51e4d3f0ce3e00718e2ff01 Mon Sep 17 00:00:00 2001
From: Max Buchholz <max.buchholz@gmx.de>
Date: Sun, 21 Apr 2024 15:17:30 +0000
Subject: [PATCH] Complement org.kde.okular.kirigami.appdata.xml

Based on what is already available in the [LinuxPhoneApps listing](https://linuxphoneapps.org/apps/org.kde.okular.kirigami.desktop/) I added some missing metainfo:
- Adds vcs-browser url
- Adds developer information
- Adds categories
---
 mobile/app/org.kde.okular.kirigami.appdata.xml | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/mobile/app/org.kde.okular.kirigami.appdata.xml b/mobile/app/org.kde.okular.kirigami.appdata.xml
index 32d1cc332c..20b7a68022 100644
--- a/mobile/app/org.kde.okular.kirigami.appdata.xml
+++ b/mobile/app/org.kde.okular.kirigami.appdata.xml
@@ -99,6 +99,10 @@
   <summary xml:lang="x-test">xxDocument Viewerxx</summary>
   <summary xml:lang="zh-CN">文档查看器</summary>
   <summary xml:lang="zh-TW">文件檢視器</summary>
+  <developer id="org.kde">
+    <name>KDE</name>
+    <url>https://kde.org</url>
+  </developer>
   <description>
     <p>
    Okular is a universal document viewer developed by KDE. Okular works on multiple platforms, including but not limited to Linux, Windows, Mac OS X, *BSD, etc.
@@ -390,8 +394,15 @@
       <image>https://cdn.kde.org/screenshots/okular/okular-mobile.png</image>
     </screenshot>
   </screenshots>
+  <categories>
+    <category>Qt</category>
+    <category>KDE</category>
+    <category>Office</category>
+    <category>Viewer</category>
+  </categories>
   <url type="homepage">https://okular.kde.org/</url>
   <url type="bugtracker">https://bugs.kde.org/enter_bug.cgi?format=guided&amp;product=okular</url>
+  <url type="vcs-browser">https://invent.kde.org/graphics/okular</url>
   <project_group>KDE</project_group>
   <provides>
     <binary>okularkirigami</binary>
-- 
GitLab

